src/schema/firefox-schemas-import.js

```diff
@@ -324,9 +347,35 @@ function loadSchemasFromFile(basePath) {
   return schemas;
 }
 
-export function importSchemas(firefoxPath, ourPath) {
+export function importSchemas(firefoxPath, ourPath, importedPath) {
   const rawSchemas = loadSchemasFromFile(firefoxPath);
   const ourSchemas = readSchema(ourPath, 'manifest.json');
   const processedSchemas = processSchemas(rawSchemas, ourSchemas);
-  writeSchemasToFile(firefoxPath, processedSchemas);
+  writeSchemasToFile(firefoxPath, importedPath, processedSchemas);
+}
+
+function downloadUrl(version) {
+  return `https://hg.mozilla.org/mozilla-central/archive/FIREFOX_AURORA_${version}_BASE.tar.gz`;
```

Comment:
As in, I don't expect it to change other than that one time so updating the source is probably the easiest and most convenient.